core/include/traccc/sanity/ordered_on.hpp

```diff
@@ -41,17 +41,16 @@ namespace traccc::host {
  * @return true If the vector is ordered on `R`.
  * @return false Otherwise.
  */
-template <std::semiregular R, typename T>
-requires std::relation<R, T, T> bool is_ordered_on(
-    R relation, vecmem::data::vector_view<T> vector) {
-    // Grab the number of elements in our vector.
-    uint32_t n = vector.size();
+template <std::semiregular R, typename CONTAINER>
```

Comment:
As before, this code doesn't have to be changed at all.

Reply:
Have a look at your functions now! They should be a lot more agreeable now that they use the `vecmem::edm::proxy` type...

Reply:
Much better, thanks!